cmd/guacone/cmd/vulnerability.go

```diff
@@ -51,7 +49,7 @@ type queryOptions struct {
 }
 
 var queryVulnCmd = &cobra.Command{
-	Use:   "vuln [flags] purl",
+	Use:   "vuln [flags] purl vulnerability ID",
```

Comment:
nit: vulnerability_id so its not seen as 2 arguments

Reply:
removed this as it is already part of the flags. It was a mistaken add.